js/adapt-contrib-bookmarking.js

```diff
@@ -97,6 +139,20 @@ class Bookmarking extends Backbone.Controller {
     });
   }
 
+  navigateTo() {
+    const locationConfig = this.config._location;
```

Comment:
```suggestion
    const locationConfig = this.config._location || 'previous';
```
Default for backward compatibility.

Reply:
This default is missing and is required for backward compatibility.